scss/_patterns_switch.scss

```diff
@@ -0,0 +1,61 @@
+@import 'settings';
+
+@mixin vf-p-switch {
+  .p-switch {
+    align-items: stretch;
+    border: 0;
+    display: inline-flex;
+    font-size: .875rem;
+    height: 2rem;
+    padding: initial;
+    position: relative;
+    width: 4rem;
+
+    &:focus {
+      outline: 1px solid $color-link;
+      outline-offset: 3px;
+    }
+
+    :first-child,
```

Comment:
Is this valid Sass without a parent (`&`) selector? Checking with a [quick codepen](https://codepen.io/barrymcgee/pen/rzraPo?editors=1100), I don't think it is?